<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DXC_Options</class>
 <widget class="QDialog" name="DXC_Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DXC Options</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>390</width>
     <height>165</height>
    </rect>
   </property>
   <property name="title">
    <string>DX Cluster Settings</string>
   </property>
   <widget class="QPushButton" name="pushButton_DXCConnect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DXCDisconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DXCAddress">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DXCPort">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>7300</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DXCSpottimeout">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DXCSpottimeout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Spot Timeout/min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DXCAddress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DXCPort">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
